bedrock-entry.sh

```diff
@@ -96,17 +92,13 @@ case ${VERSION^^} in
     ;;
   *)
     # use the given version exactly
+    echo "Using given version ${VERSION}"
+    lookupVersion serverBedrockLinux "${VERSION}"
```

Comment:
@itzg I can change this to check if `${PREVIEW}` is true, then pass in `serverBedrockLinuxPreview` instead so that we don't lose the functionality you highlighted below.